app/modules/mutations/createReferenceModule.ts

```diff
@@ -10,7 +10,11 @@ function capitalizeFirstLetter(string) {
   return string.charAt(0).toUpperCase() + string.slice(1)
 }
 
-export default resolver.pipe(resolver.authorize(), async ({ doi }, ctx) => {
+interface Input {
+  doi: string
+}
+
```

Comment:
I get the improvement itself, but please keep PRs related to the topic (it can be very distracting as I know it does not relate here, which is not always clear).